### PR TITLE
widget redesign

### PR DESCRIPTION
**Pull request by Asergey91**

### CODE REVIEW

**Review by Dietr — APPROVED** [review]

File: tests/integration/components/standard-template/template-card-test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { module, test } from 'qunit';
```
Comment (nvdk) [inline tests/integration/components/standard-template/template-card-test.js]: remove default tests

**Comment by benjay10** [comment]
I don't have an "Approve" button, but I approve anyway. :)

Please also add a label to this PR!

**Comment by lagartoverde** [comment]
Ok things to consider
- Maybe when there are no items in the menu we should hide it?
- Maybe we should give the user some feedback that he has inserted something? closing the menu or anything
- The insert links also should have the cursor as a pointer and the hover property as we have on the invoegen one

**Review by nvdk — CHANGES_REQUESTED** [review]
this seems to be a bit of a regression. I can't insert templates into an empty document?

**Pull request merged**